docker/README.md

```diff
@@ -7,11 +7,14 @@ Initial setup
    * Install [Docker](https://docs.docker.com/engine/installation/)
    * Install [Docker Compose](https://docs.docker.com/compose/install/) (Note you can also install in a virtualenv with `$ pip install docker-compose`)
 * OS X
-   * Install [Docker Toolbox](https://docs.docker.com/toolbox/toolbox_install_mac/). Go through the full tutorial, which will create a default machine.
-   * To create a new VM manually, run `docker-machine create default --driver=virtualbox` (not necessary if you followed the Docker Toolbox tutorial).
-   * If not using the Quick Start terminal, run `eval $(docker-machine env default)` to set up Docker's environment variables.
+   * Preferred: install [Docker for Mac](https://docs.docker.com/docker-for-mac/install/).
+   * Alternate for older Macs that do not support Docker for Mac:
+     * Install [Docker Toolbox](https://docs.docker.com/toolbox/toolbox_install_mac/). Go through the full tutorial, which will create a default machine.
+     * To create a new VM manually, run `docker-machine create default --driver=virtualbox` (not necessary if you followed the Docker Toolbox tutorial).
```

Comment:
Is it intentional that these 2 steps (`To create a new VM manually...` and `If not using the Quick Start terminal...`) are sub-steps of using Docker Toolbox? i.e. are they in fact only needed when running Docker Toolbox, not Docker for Mac (I did these 2 steps and they ran fine/didn't do any harm, but they could very well have been unnecessary)

Reply:
they are only necessary for Docker Toolbox. `docker-machine` is part of Docker Toolbox, and there is no equivalent (that I know of) for Docker for Mac